proposals/templates/proposals/proposal_diff.html

```diff
@@ -65,6 +65,29 @@ <h2 class="mt-3 mb-1">{% trans "Algemene informatie over de aanvraag" %}</h2>
                             <td>{{ proposal.supervisor.get_full_name }}</td>
                         </tr>
                     {% endif %}
+                    {%  if proposal.relation.check_in_course or p_proposal.relation.check_in_course%}
+                    <tr>
+                        <th>{% get_verbose_field_name "proposals" "proposal" "student_program" %}</th>
+                        <td>{{ p_proposal.student_program }}</td>
+                        <td>{{ proposal.student_program }}</td>
+                    </tr>
+                    <tr>
+                        <th>{% get_verbose_field_name "proposals" "proposal" "student_context" %}</th>
+                        <td>{{ p_proposal.student_context }}</td>
```

Comment:
Edge case: if unselected in the previous revision, `p_proposal.student_context` will display an untranslated `"None"`, it would be nicer if this just displayed an empty string.

This can be done with a filter, or maybe with `get_FIELDNAME_display()` because it's a choice (not sure about the latter).

https://docs.djangoproject.com/en/4.1/ref/models/instances/#django.db.models.Model.get_FOO_display